config/checkstyle/checkstyle.xml

```diff
@@ -111,7 +111,7 @@
             <property name="sortImportsInGroupAlphabetically" value="true"/>
         </module>
         <module name="CyclomaticComplexity">
-            <property name="max" value="12"/>
```

Comment:
We should just remove these two rules as we are just incrementing the value as we need.